services-camel-app/src/main/java/edu/unc/lib/boxc/services/camel/streaming/StreamingPropertiesRequestProcessor.java

```diff
@@ -67,15 +69,18 @@ private String validate(StreamingPropertiesRequest request, PID pid) {
 
         if (Objects.equals(ADD, action)) {
             var url = request.getUrl();
+            var type = request.getType();
             if (url == null) {
                 return "URL is required";
             } else {
                 if (!url.startsWith(STREAMREAPER_PREFIX_URL)) {
                     return "URL is not a stream reaper URL";
                 }
             }
-            if (request.getType() == null) {
+            if (type == null) {
                 return "Streaming type is required";
+            } else if (!type.equals(STREAMREAPER_SOUND) && !type.equals(STREAMREAPER_VIDEO)) {
+                return "Streaming type is not a valid stream reaper type";
```

Comment:
"Streaming type is not a valid type" so we aren't as tightly coupled with stream reaper.